package.json

```diff
@@ -47,32 +45,38 @@
         "lines": 100,
         "statements": 100
       }
-    }
+    },
+    "preset": "ts-jest"
```

Comment:
maybe will be better to use @swc/jest ?

Reply:
The real benefit of using ts-jest is that it do typechecking in test. 
For projects I would suggest speed is more important, but for package with 10 test it won't have such an effect, but with ts-jest we get extra coverage